docs/Project.toml

```diff
@@ -10,6 +10,7 @@ HybridArrays = "1baab800-613f-4b0a-84e4-9cd3431bfbb9"
 IJulia = "7073ff75-c697-5162-941a-fcdaad2a7d2a"
 Manifolds = "1cead3c2-87b3-11e9-0ccd-23c62b72b94e"
 ManifoldsBase = "3362f125-f0bb-47a3-aa74-596ffd7ef2fb"
+MultivariateStats = "6f286f6a-111f-5878-ab1e-185364afe411"
```

Comment:
I think this should be added to the `tutorials` environment and not to the docs (yes yes we have 3 environments by now, but I think its better to split that).

Reply:
Ah and CSV seems to be missing from the `tutorials/` project toml as well it seems.

Reply:
I see, I was confused where to put that.

Reply:
No problem. The idea is that the quarto part is a bit encapsulated besides the docs. I am not even 100% sure IJulia is needed in the docs, it should not.
Splitting docs and tutorials maybe makes the two environments a bit nicer themselves, less dependencies.